script.js

```diff
@@ -251,9 +259,19 @@ L.tileLayer(
  * URI HELPERS
  *****************************************/
 
+function createCitiesCartoURI() {
```

Comment:
@jimboweb try this query:

```sql
SELECT
municipality, range, policy_type, policy_summary, link,
lat, lng as lon,
CASE passed WHEN true THEN 'Yes' ELSE 'No' END as passed
FROM public.emergency_tenant_protections_scored
WHERE admin_scale = 'City'
```

Reply:
@jimboweb also make sure to not hardcode the protections table name in the query, use the `cartoSheetSyncTable` variable.